Source/Obsolete/MockFactory.cs

```diff
@@ -330,25 +329,23 @@ protected virtual void VerifyMocks(Action<Mock> verifyAction)
 		{
 			Guard.NotNull(verifyAction, nameof(verifyAction));
 
-			var message = new StringBuilder();
+			var failures = new List<UnmatchedSetups>();
 
 			foreach (var mock in mocks)
 			{
 				try
 				{
 					verifyAction(mock);
 				}
-				catch (MockVerificationException mve)
+				catch (MockException ex) when (ex.Error is UnmatchedSetups failure)
```

Comment:
Unfortunately the throw-catch pattern cannot be replaced with a full try pattern due to this method's signature and the fact that it's `protected`. It would have to be marked as `[Obsolete]`.